common/app/views/fragments/page/head/stylesheets/styles.scala.html

```diff
@@ -52,6 +54,7 @@
 <!--[if (gt IE 9)|(IEMobile)]><!-->
 @styles.criticalCss
 @styles.linkCss
+@styles.footerCss
```

Comment:
Should these styles go outside the IE conditional statements? It looks like we'll need them on every browser?

Reply:
This final block of 3 `@styles` links targets all browsers, the 2 blocks above are targeting IE.